components/03-forms/10-misc/dropdown-button/dropdown-button.config.js

```diff
@@ -17,25 +17,28 @@ module.exports = {
       context: {
         id: 'actie_open',
         open: true,
-        items:
-          [
-            [
+        items: [
+          {
+            group: [
```

Comment:
De changes in deze file zijn niet nodig, graag terugzetten en template op het oude model hangen

Reply:
Snap ik niet?

Reply:
Samen opgelost